.gas-snapshot

```diff
@@ -12,34 +12,35 @@ LockedNORILib_availableAmount:test() (gas: 12340)
 Market_ALLOWLIST_ROLE:test() (gas: 12702)
 Market_setPriorityRestrictedThreshold:test() (gas: 154655)
 Market_setPriorityRestrictedThreshold:test_zeroAvailable() (gas: 149660)
-Market_withdraw_1x3_center:test() (gas: 325057)
-Market_withdraw_2x1_back:test() (gas: 327029)
-Market_withdraw_2x1_front:test() (gas: 316326)
-Market_withdraw_2x1_front_relist:test() (gas: 369967)
-Market_withdraw_as_DEFAULT_ADMIN_ROLE:test() (gas: 263259)
-Market_withdraw_as_operator:test() (gas: 272253)
-Market_withdraw_as_supplier:test() (gas: 261415)
+Market_withdraw_1x3_center:test() (gas: 332480)
+Market_withdraw_2x1_back:test() (gas: 334452)
+Market_withdraw_2x1_front:test() (gas: 323749)
+Market_withdraw_2x1_front_relist:test() (gas: 377393)
+Market_withdraw_as_DEFAULT_ADMIN_ROLE:test() (gas: 265882)
+Market_withdraw_as_operator:test() (gas: 274876)
+Market_withdraw_as_supplier:test() (gas: 264038)
 Market_withdraw_reverts:test() (gas: 135794)
-Removal__beforeTokenTransfer:test() (gas: 5311)
+Removal__beforeTokenTransfer:test() (gas: 7593)
 Removal__beforeTokenTransfer:test_paused_reverts_Paused() (gas: 29426)
-Removal__beforeTokenTransfer:test_zeroValueTransfer_reverts_RemovalAmountZero() (gas: 8938)
-Removal_mintBatch:test() (gas: 429101)
-Removal_multicall:test_balanceOfBatch() (gas: 598147)
+Removal__beforeTokenTransfer:test_zeroValueTransferToMarket_reverts_RemovalAmountZero() (gas: 8960)
+Removal_market_balance:test_market_balance() (gas: 1281480)
```

Comment:
also FYI-- this test provides a high value because it does all the test setup within the `test_market_balance` fn instead of `setUp()`

Reply:
yeah, I could maybe move one bit of setup outside of this test but it basically relies on assertions happening in between making changes like listing, selling, withdrawing, so I can't really extract everything cleanly